var/spack/repos/builtin/packages/py-gitpython/package.py

```diff
@@ -10,19 +10,23 @@ class PyGitpython(PythonPackage):
     """GitPython is a python library used to interact with Git repositories."""
 
     homepage = "http://gitpython.readthedocs.org"
-    pypi = "gitpython/GitPython-3.1.7.tar.gz"
+    pypi = "GitPython/GitPython-3.1.12.tar.gz"
```

Comment:
```suggestion
    pypi = "gitpython/GitPython-3.1.12.tar.gz"
```
CamelCase doesn't work for me